protected/config/env.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$env = [];
```

Comment:
If I understand it correctly, this file should transfer ENV variables to the config file. I like this.

But lets move the file to `/protected/humhub/config`, because 
- /protected/config (local configuration)
- /protected/humhub/config (base configuration, not changeable)